csi/server/deploy/kubernetes/csi-configmap-opensdsplugin.yaml

```diff
@@ -12,7 +12,7 @@ data:
   osusername: admin
   ospassword: opensds@123
   passwordencrypter: aes
-  enableEncrypted: false
+  enableEncrypted: F
```

Comment:
`F` is not human readable, please use `false`. `enableEncrypted` also should be changed

Reply:
Already been modified. Please review again.